src/Tribe/Views/V2/View.php

```diff
@@ -456,7 +455,8 @@ public static function make( $view = null, Context $context = null ) {
 
 		// This code is coupled with the idea of viewing events: that's fine as Events are the default view content.
 		$view_repository = tribe_events();
-		$view_repository->order_by( 'event_date', 'ASC' );
+		// Sort events  by start date first and by duration second, this is equivalent to sorting them by end date.
+		$view_repository->order_by( [ 'event_date' => 'ASC', 'event_duration' => 'ASC' ] );
```

Comment:
:no_entry_sign: When a multi-item array uses associative keys, each value should start on a new line.

<sub>hash: 13be3b367c78982e6f24f81dc0f302c5</sub>